### ISSUE TITLE
There are html tags left after user empty the editor

### ISSUE THREAD

**Issue by PikaboyZ** [davidroyer/vue2-editor#25]
After user remove everything from the editor, there is `<p><br></p>` left.

Should it be empty? Or you have some reasons for this?

**Comment by davidroyer** [comment]
I believe this is just part of the Quill.js functionality. This is not something I built it. Have you found this to cause any issues for you?

**Comment by lemar5555** [comment]
The same issue,
making harder to validate on backend, writing rule required|string is passing without a problem, but in shouldn't as it only contains empty tags not the content itself.

**Comment by ylethe** [comment]
I have meet the same issue, it cause a bug to me.please fix it if author have time.

**Comment by SlyNet** [comment]
Hard to use component, since setting a `v-model` value to `null` anyway makes the value to be `<p><br/></p>`

**Comment by MellowCobra** [comment]
I've run into this issue as well. Turns out, it is an intentional behavior of the underlying Quill functionality.

It has been discussed at [Quill issue 1235](https://github.com/quilljs/quill/issues/1235).
The tl;dr is that some basic functionality of Quill assumes at least a block and `inline blot` in order to function properly. Most users have resigned to take it upon themselves to calculate whether or not the Quill form is empty themselves.

I am hoping that a Quill-native solution is added in the future. Then perhaps @davidroyer could add a Vue hook for us ;)

**Comment by Mushood** [comment]
If anyone else is using VeeValidate: https://vee-validate.logaretm.com/

Here is how i countered the issue:

I added an additional rule like so:
```
this.$validator.extend('not_empty', {
                getMessage: field => 'My message',
                validate: value => this.check(),
            });
```
On my model

`<vue-editor v-model="body"  v-validate="'required|description_empty'"></vue-editor>`

The check function

```
            check: function(){
                const vm = this;
                if(vm.strip(vm.session.body) == "" ){

                    return false;
                }

                return true;
            },
```
and the corresponding strip function

```
            strip: function(html){
                var doc = new DOMParser().parseFromString(html, 'text/html');
                return doc.body.textContent || "";
            },

```

Hope this helps you some hours

**Comment by davidroyer** [comment]
So I just published 2.4.3 which includes a fix for this issue.

I'm going to leave this issue open for a week and to see if there are any other related issued experienced by anyone.

**Comment by PedroLopesMV** [comment]
Hello David. I was testing the 2.4.3 that's just been released and i saw that the fix that you implemented only works if the user is writing using only the paragraph option. Let me exemplify some examples of the "bugs":

- If the user select one of the following options (Bold,Italic or Underlined), gives two spaces on the editor and then erases the content there are leftovers tags in the editor.

- Another case happens when the user changes the text from paragraph to one of the headings types. In this case, even before the user types anything, the editor already places the Heading tag with a <br> in the middle. 

I've been trying to fix the problem on my own, but i have not been successful. I'd aprecciate some help on this issue if possible. Thanks in advance.
